package-system/zlib/build_zlib_linux.sh

```diff
@@ -6,14 +6,11 @@
 #
 #
 
-cmake -S temp/src -B temp/build -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -DCMAKE_C_FLAGS=-fPIC -DSKIP_INSTALL_FILES=YES
-if [ $? -ne 0 ]; then
-    echo "Error generating build"
-    exit 1
-fi
+cmake -S temp/src -B temp/build \
+     -DCMAKE_BUILD_TYPE=Release \
+     -DCMAKE_CXX_STANDARD=17 \
+     -DCMAKE_C_FLAGS=-fPIC \
+     -DSKIP_INSTALL_FILES=YES || exit 1
```

Comment:
the default generator for Linux is `Unix Makefiles`, I suggest using Ninja instead (which we already use for Android) 
For this lib may not be a big difference, but if others are going to copy this pattern and build a big library, it will.

Reply:
Ok, wlil switch to ninja.  I take it if you dont _have_ ninja, CMAKE outputs something useful.

Reply:
Yes CMake will output that it can't find ninja during generation.

Reply:
Done!  Didn't publish a new package, but did rerun the build script.